src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs

```diff
@@ -21,22 +21,34 @@ namespace NuGet.CommandLine.XPlat
     internal class ListPackageCommandRunner : IListPackageCommandRunner
     {
         private const string ProjectAssetsFile = "ProjectAssetsFile";
-        private const string ProjectName = "MSBuildProjectName";
+        private const int GenericSuccessExitCode = 0;
+        private const int GenericFailureExitCode = 1;
         private Dictionary<PackageSource, SourceRepository> _sourceRepositoryCache;
 
         public ListPackageCommandRunner()
         {
             _sourceRepositoryCache = new Dictionary<PackageSource, SourceRepository>();
         }
 
-        public async Task ExecuteCommandAsync(ListPackageArgs listPackageArgs)
+        public async Task<int> ExecuteCommandAsync(ListPackageArgs listPackageArgs)
         {
+            IReportRenderer reportRenderer = listPackageArgs.Renderer;
+            (int exitCode, ListPackageReportModel reportModel) = await GetReportDataAsync(listPackageArgs);
```

Comment:
Data model is acquired here, it's shared for console/json output.

Reply:
Can we resolve this too?

I'm having a tough time knowing what's actually something being worked vs what's actually a review helper. 
IMO this isn't helpful for reviewing. The code is pretty self descriptive.

Reply:
Yes, you can resolve. Sorry, I thought having extra comment would help.

Reply:
I understand the intention. Maybe others find it helpful, so it's ok to have it. Personally my logic is if when I review, there are no active threads, then I'm reviewing the current change, not any potential new changes. That's trickier, so that info is super valuable.